viewer/src/utils/CogniteClient3dExtensions.ts

```diff
@@ -77,9 +78,14 @@ export class CogniteClient3dExtensions {
   }
 
   public async retrieveBinaryBlob(blobId: number, path?: string): Promise<ArrayBuffer> {
-    const url = this.buildBlobBaseUrl(blobId) + (path ? `/${path}` : '');
-    const response = await this.client.get<ArrayBuffer>(url);
-    return response.data;
+    const url = this.client.getBaseUrl() + this.buildBlobBaseUrl(blobId) + (path ? `/${path}` : '');
```

Comment:
url's should be using string templates and named functions can be made separate.. requestWithPath, requestWithoutPath